perf/perftool.py

```diff
@@ -57,10 +66,21 @@ def test(self):
             if '-- [ FAIL ] --' in line:
                 self.count += 1
                 self.log.info(line)
-
         if self.count > 0:
             self.fail("%s Test failed" % self.count)
 
+        process.system_output("perf test")
```

Comment:
process is not defined any where, please add import statement
from avocado.utils import process